src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -740,4 +740,274 @@
   <data name="OpenCWDCommandKey" xml:space="preserve">
     <value>Open current working directory</value>
   </data>
+  <data name="CopyText" xml:space="preserve">

```

Comment:
Lightly pushing back on the capitalization of these. We should probably capitalize the first letter of the first word and that's it.